src/components/Availability.test.js

```diff
@@ -247,17 +248,18 @@ it("shows total slots across all available days", async () => {
                     appointmentData: {
                         "10/11/2021": {
                             hasAvailability: true,
-                            numberAvailableAppointments: 35,
+                            numberAvailableAppointments: 34,
```

Comment:
I assume you tweaked these numbers just to add a little variation.

Reply:
Yes.  I wanted to make sure that it wasn't counting the first row twice.